index.js

```diff
@@ -23,10 +23,12 @@ var https = 'https://'
 function license(options) {
   var settings = options || {}
   var finals = settings.ignoreFinalDefinitions
+  var test = settings.heading || licenseHeadingRegexp
+
   var headingOptions = {
     ignoreFinalDefinitions:
       finals === undefined || finals === null ? true : finals,
-    test: licenseHeadingRegexp
+    test
```

Comment:
```suggestion
    test: test
```

Just to be careful and not break anyone for now

Reply:
Alright, committed it!